xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/Wrappers.java

```diff
@@ -267,6 +269,38 @@ public boolean equals(Object obj) {
     }
   }
 
+  /**
+   * A wrapped {@link String} containing the Uri.
+   */
+  @Value.Immutable
+  @Wrapped
+  @JsonSerialize(as = Uri.class)
+  @JsonDeserialize(as = Uri.class)
+  abstract static class _Uri extends Wrapper<String> implements Serializable {
```

Comment:
What do you think about renaming this to `NfTokenUri` so it's clear that it's for NFTs?

Reply:
Yes, absolutely. Gives more context.